tsconfig.client.json

```diff
@@ -0,0 +1,11 @@
+{
+	"compilerOptions": {
+		"types": [
+			"node"
+		]
+	},
+	"include": [
+		// "tsup.config.client.ts",
```

Comment:
This is invalid json. Remove if not used?

Reply:
It's actually not invalid. But yeah I know ye're allergic to comments, so I can remove it :)